### ISSUE TITLE
Trailing "✓" is ISOSegmentValidator error report

### ISSUE THREAD

**Issue by rbouqueau** [Dash-Industry-Forum/DASH-IF-Conformance#667]
When validating e.g. https://dash.akamaized.net/WAVE/vectors/chh1_sets/12.5_25_50/t1/2023-05-12/stream.mpd, the details show:

```
✗ ### error: moov-1:trak-1:mdia-1:minf-1:stbl-1:stsd-1
### ../src/ValidateBitStreams.cpp : 2965 : ### error: moov-1:trak-1:mdia-1:minf-1:stbl-1:stsd-1
### Validate_NAL_Unit_HEVC: Trailing zero bits not zero 1
✗ ### error: moov-1:trak-1:mdia-1:minf-1:stbl-1:stsd-1
### ../src/ValidateBitStreams.cpp : 3219 : ### error: moov-1:trak-1:mdia-1:minf-1:stbl-1:stsd-1
### Validate_NAL_Unit_HEVC: Trailing zero bits not zero 1
! ../src/ValidateAtoms.cpp : 2400 : Warning: In moov-1:trak-1:mdia-1:minf-1:stbl-1:stsd-1 - unknown atom found "btrt": video sample descriptions would not normally contain this
! ../src/ValidateAtoms.cpp : 2400 : Warning: In moov-1:trak-1:mdia-1:minf-1:stbl-1:stsd-1 - unknown atom found "pasp": video sample descriptions would not normally contain this
✓ ../src/ValidateAtoms.cpp : 3999 : colr atom of type nclx found, the software does not handle colr atoms of this type.
! ../src/ValidateAtomList.cpp : 2339 : WARNING: In moov-1:udta-1 - unknown/unexpected atom 'meta'
✗ ### error:
### ../src/PostprocessData.cpp : 1147 : ### error:
### According to DASH-IF IOP Section 3.2.8 @bandwidth of the Representation (3500000 bps) is set too low given the @minimumBufferTime (2 s), the minimum @bandwidth value required to conform is 3716606 bps.
✓ 
```

What is the trailing "✓" about?

**Comment by rbouqueau** [comment]
Does anyone know?

**Comment by dsilhavy** [comment]
Fixed in #683 . This was caused by an empty line in the segment validator output.
